test/dialect/postgresql/test_types.py

```diff
@@ -8,6 +8,11 @@
 import re
 import uuid
 
+try:
+    from asyncpg import BitString
```

Comment:
so can you give me the background on this part?    Having to use an asyncpg import is...bad.   it only recognizes this exact type and not a simple 1/0 ?

Reply:
I was assuming so reading through asyncpg's docs, but I dug deeper and realized it also accepts a byte value. I'll change the code accordingly to prevent importing from asyncpg.